### PR TITLE
[CDAP-5792] [CDAP-5799] hydrator improvements

### PR DESCRIPTION
**Pull request by elfenheart**
- Reduce padding on plugin list (to accommodate longer plugin name)
- Modify ACE Editor to have tab size 2
- DAG connections in studio view should be shorter
- Fix fit to screen function to make sure the DAG is properly centered (it was offsetting to the right before)
- Modal header for Templates Gallery should be consistent
- Data Pipeline List view

### CODE REVIEW

File: cdap-ui/app/directives/widget-container/widget-js-editor/widget-js-editor.js [hunk 1]
```diff
@@ -31,9 +31,14 @@ angular.module(PKG.name + '.commons')
           workerPath: '/assets/bundle/ace-editor-worker-scripts',
           mode: $scope.mode || 'javascript',
           useWrapMode: true,
-          newLineMode: 'unix'
+          newLineMode: 'unix',
+          onLoad: aceLoaded
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/widget-container/widget-js-editor/widget-js-editor.js]: I think we can do this (as specified by a workaround in angular ui ace editor [Workaround](https://github.com/angular-ui/ui-ace/blob/master/src/ui-ace.js#L100) 

```
$scope.aceoptions = {
   ...,
   newLineMode: 'unix',
   advanced: {
     tabSize: 2
   }
}
```

File: cdap-ui/app/features/hydratorplusplus/services/hydrator-service.js [hunk 2]
```diff
@@ -49,7 +49,13 @@ class HydratorPlusPlusHydratorService {
     });
     connections = config.connections;
     // Obtaining layout of graph with Dagre
-    var graph = this.DAGPlusPlusFactory.getGraphLayout(nodes, connections);
+    var graph;
+    if (isStudio) {
+      graph = this.DAGPlusPlusFactory.getGraphLayout(nodes, connections, 100);
```
Comment (ajainarayanan) [inline cdap-ui/app/features/hydratorplusplus/services/hydrator-service.js]: Is there any specific reason why we are not having this for both studio and detailed view ?

Comment (elfenheart) [inline cdap-ui/app/features/hydratorplusplus/services/hydrator-service.js]: detailed view, we have to preserve the space for metric on DAG

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by elfenheart** [comment]
demoed to @sreevatsanraman

**Pull request merged**